qiskit/ignis/verification/entanglement/parallelize.py

```diff
@@ -0,0 +1,449 @@
+
+
+'''
+In this package, there are three modules, linear.py,
+parallellize.py, and analysis.py. It may be more suitable
+to put these in Terra rather than Ignis. The most
+important module, parallellize.py provides methods to
+parallellize CNOT gates in the preparation of the GHZ State,
+which results in the GHZ State having a much higher fidelity
+then a normal "linear" CNOT gate preparation of the
+GHZ State. Additionally, there are methods within parallelize.py
+that configure different characterization tests for the
+GHZ State, including Multiple Quantum Coherence (MQC),
+Parity Oscillations (PO), and Quantum Tomography.
+'''
+
+from qiskit import *
+from qiskit.circuit import Parameter
+
+
+class BConfig:
+    '''
+    This class is used to create a GHZ circuit
+    with parallellized CNOT gates to increase fidelity
+    '''
+
+    def __init__(self, backend, indicator=True):
+        self.nodes = {}
+        self.backend = backend
+        self.cmap = backend.configuration().coupling_map
+        self.add_nodes()
+        self.indicator = indicator
+
+    def add_nodes(self):
+        '''
+        Adds nodes to the dictionary based on coupling map
+        '''
+        self.nodes = {}
```

Comment:
"add" usually gives the feeling that you add to something that already exists; here, rather, you initialize.

Reply:
right, changed to initialize